vagrant/provisioning/roles/confluent-platform-install/templates/confluent-kafka.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Apache Kafka - broker
+Documentation=http://docs.confluent.io/
+After=network.target confluent-zookeeper.target
+
+[Service]
+Type=simple
+User=cp-kafka
+Group=confluent
+Environment="LOG_DIR=/opt/arkcase/log/kafka"
+ExecStart=/usr/bin/kafka-server-start /etc/kafka/server.properties
+ExecStop=/opt/kafka/bin/kafka-server-stop.sh
```

Comment:
This should cause the Kafka application log files to be written to {{root_folder}}/log/kafka, but we still need configuration to cause the data files to go into {{root_folder}}/data/kafka.  By default they will go into /tmp which definitely we don't ant.